docs/source/tutorials/QUBO_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>where does this matrix come from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>It is a randomly generated symmetric matrix. I used it here as a general but (more) complicated QUBO-problem matrix.</p>